packages/fluentui/react-northstar-prototypes/src/prototypes/FormValidation/Formik.tsx

```diff
@@ -1,61 +0,0 @@
-import * as React from 'react';
-import { Form } from '@fluentui/react-northstar';
-import { Formik } from 'formik';
-
-const FormValidateFormik = () => (
-  <Formik
-    initialValues={{
-      firstName: '',
-      city: '',
-    }}
-    validate={values => {
-      const errors: {
-        firstName?: string;
-        city?: string;
-      } = {};
-      if (!values.firstName) {
-        errors.firstName = 'Required';
-      }
-      if (values.firstName && values.firstName.length < 4) {
-        errors.firstName = 'Too small';
-      }
-      if (!values.city.length) {
-        errors.city = 'Required';
-      }
-      return errors;
-    }}
-    onSubmit={(values, { setSubmitting }) => {
-      // eslint-disable-next-line no-console
-      console.log(JSON.stringify(values, null, 2));
-      setSubmitting(false);
-    }}
-    render={({ handleSubmit, handleChange, errors, handleBlur, setTouched, touched, values, setFieldValue }) => (
-      <Form onSubmit={handleSubmit}>
-        <Form.Input
-          errorMessage={touched.firstName && errors.firstName}
-          onBlur={handleBlur}
-          onChange={handleChange}
-          showSuccessIndicator={Boolean(values.firstName.length >= 4)}
-          label="First Name"
-          name="firstName"
-          id="first-name-inline"
-        />
-        <Form.Dropdown
-          onBlur={e => {
-            setTouched({ ...touched, city: true });
-          }}
-          onChange={(e, props) => {
-            setFieldValue('city', props.value);
-          }}
-          label="City"
-          id="city"
-          errorMessage={touched.city && errors.city}
-          items={['prague', 'new york']}
-        />
-        <Form.Button content="Submit" />
-      </Form>
-    )}
-  />
-);
-
-export default FormValidateFormik;
```

Comment:
I don't think that it's an expected change... 🙀

Reply:
Some background: we ran into a major issue yesterday in the update;

Formik depends on `scheduler`; however, we found after the update the new webpack complains about imports of "LowPriority", which doesn't export (only `unsafe_LowPriority`.) I am not sure how this worked before, because looking at the source of scheduler it does not seem like LowPriority has been an export.

We tried updating formik, which fixes this issue in later builds and removes the scheduler dependency. Unfortunately our version of TypeScript is not compatible with its d.ts content as we get build errors doing so.

Given we only use Formik in 1 example in 1 prototype, it seems best to remove the example, than to try and also upgrade TypeScript.

If you think this is important to keep, we might try and find a version of formik that's compatible with our TypeScript AND removes the scheduler dependency. But there have been so many little problems that this seemed like the safest option.

Reply:
we need to have the example - but maybe we can just create a codesandbox with formik and put a link to the prototypes page?

Reply:
Alright; let me first see if there's a version available that doesn't have the scheduler dep and also is compatible with our TS version; otherwise I can do the iframe workaround.

Reply:
done! updated to a newer version.